src/main/java/gov/nasa/pds/tools/validate/rule/pds4/DataDefinitionAndContentValidationRule.java

```diff
@@ -114,7 +114,8 @@ public void validate() throws MalformedURLException, URISyntaxException {
           getTarget(), objectCounter, -1));
       // e.printStackTrace();
     } finally {
-      label.close();
+        if(nonNull(label))
```

Comment:
This doesn't compile for me (jdk1.8).

Need to add `import java.util.Objects;` to the top of the file and change the conditional to:
```java
if (Objects.nonNull(label))
```

Reply:
My bad, @nutjob4life - thanks for the saving catch

Reply:
I thought we were supposed to use JDK 11 or 15 or something, according to docs. Nonetheless, yes, it was missing the import.